
uint8 LEFT=0
uint8 RIGHT=1

uint8 FREQ_ON=255
uint8 FREQ_CRUSH=1
uint8 FREQ_SLIP=5
uint8 FREQ_OFF=0

Header header

int32[2] freq
int32[2] amp
